catalog/imaging/catalog-imaging-service-api/src/main/java/com/connexta/alliance/imaging/chip/service/api/ChipOutOfBoundsException.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Connexta, LLC
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.alliance.imaging.chip.service.api;
+
+/**
+ * A custom exception that is potentially thrown by the ChipService.
+ */
+public class ChipOutOfBoundsException extends Exception {
+
+    /**
+     *
+     * @param message a descriptive message of the reason the exception is being thrown.
+     */
+    public ChipOutOfBoundsException(String message) {
```

Comment:
As an unchecked exception, IllegalArgument is indicative of a contract violation.  Since the user will be drawing their polygon on the screen, it will be difficult to check for the out of bounds exception before submitting the request.  Arguably, there's no reason to ever used checked exceptions, but if we're going to use them then this is a good use-case.